app/views/MessagesView/index.js

```diff
@@ -46,6 +42,7 @@ class MessagesView extends React.Component {
 
 	componentDidMount() {
 		this.load();
+		this.setHeader();
```

Comment:
Should be called on constructor or maybe can cause a header blink.

Reply:
Right, I called this in the constructor now.